src/sql/workbench/contrib/notebook/browser/models/cell.ts

```diff
@@ -285,7 +285,8 @@ export class CellModel implements ICellModel {
 
 	private notifyExecutionComplete(): void {
 		if (this._notebookService) {
-			this._notebookService.serializeNotebookStateChange(this.notebookModel.notebookUri, NotebookChangeType.CellExecuted, this);
+			this._notebookService.serializeNotebookStateChange(this.notebookModel.notebookUri, NotebookChangeType.CellExecuted, this)
+				.catch(() => { });
```

Comment:
Suggest using onUnexpectedError for these to at least log the error to console - unless you think this is expected to fail and would just be clutter